mm2src/mm2_main/src/lp_swap/saved_swap.rs

```diff
@@ -376,7 +411,11 @@ mod wasm_impl {
 
     #[async_trait]
     impl SavedSwapIo for SavedSwap {
-        async fn load_my_swap_from_db(ctx: &MmArc, uuid: Uuid) -> SavedSwapResult<Option<SavedSwap>> {
+        async fn load_my_swap_from_db(
+            ctx: &MmArc,
+            _optional_address_dir: Option<&str>,
```

Comment:
nit: It's crystal clear already that it's an optional value

```suggestion
            _address_dir: Option<&str>,
```

Reply:
[212d188](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2398/commits/212d18847c8f5132b8953432d7c53bc4debcc09c)

Reply:
this is pretty simple i will resolve myself.